### PR TITLE
add collections shortcut

### PR DESCRIPTION
**Pull request by jamesgorrie**
The hardest bit about getting people to test the collections is the bookmarklet.
The now updates localStorage by pressing `ctrl+c`. We'll remove this once we're happy to make it default.

### CODE REVIEW

File: kahuna/public/js/search/index.js [hunk 1]
```diff
@@ -88,11 +92,25 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
...
+                }
+            });
         }],
         resolve: {
             shortcutKeys: [function() {
                 // keep the shortcut keys here to stop overriding
-                return new Map([['metadataPanel', 'm']]);
+                return new Map([
+                    ['metadataPanel', 'm'],
+                    ['showCollections', 'ctrl+c']
```
Comment (theefer) [inline kahuna/public/js/search/index.js]: Is there not a risk that this conflicts with the standard "Copy" shortcut (eg copying text)? Is the assumption that people are all on Mac and Ctrl+C != Cmd+C?

**Comment by theefer** [comment]
Small concern about shortcut, else :+1:

File: kahuna/public/js/search/index.js (hunk 1 above)
Comment (akash1810) [inline kahuna/public/js/search/index.js]: [Alternative shortcut](https://en.wikipedia.org/wiki/Konami_Code).

File: kahuna/public/js/search/index.js [hunk 2]
```diff
@@ -88,11 +92,25 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
...
                 callback: panels.metadataPanel.toggleHidden
             });
+
+            keyboardShortcut.bindTo($scope).add({
+                combo: shortcutKeys.get('showCollections'),
+                description: 'Toggle collections',
+                callback: () => {
+                    const toggled = !storage.getJs('showCollectionsPanel');
+                    storage.setJs('showCollectionsPanel', toggled);
+                    $window.alert('Now refresh to start using collections');
```
Comment (akash1810) [inline kahuna/public/js/search/index.js]: any reason we don't do this automatically?

Comment (jamesgorrie) [inline kahuna/public/js/search/index.js]: They might be on a search they don't want to lose. As this is hyper-temporary, I don't really mind asking.

Comment (akash1810) [inline kahuna/public/js/search/index.js]: makes sense :+1:

File: kahuna/public/js/search/index.js (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/search/index.js]: Any problems with `ctrl+alt+c`?

**Comment by theefer** [comment]
:+1:

**Pull request merged**